census_view.hpp

```diff
@@ -131,8 +131,10 @@ class CensusView final
 
     std::shared_ptr<Ledger const> composite_ledger_;
 
-    wxDataViewCtrl* list_window_;
-    wxObjectDataPtr<CensusViewDataViewModel> list_model_;
+    std::vector<int> visible_columns_;
```

Comment:
Interesting, I guess we didn't need this one with wxDVC because for it the control columns are not the same as model columns? I wonder if there could be any advantage to making this part of the model here too (i.e. the table)?